package.json

```diff
@@ -79,6 +79,7 @@
     "body-parser": "^1.15.2",
     "chai": "^4.1.1",
     "chai-as-promised": "^7.1.1",
+    "chokidar": "^2.0.2",
```

Comment:
[Everything](https://github.com/paulmillr/chokidar) uses this for file watching.